lrpar/cttests/src/multiparam.test

```diff
@@ -0,0 +1,27 @@
+name: Test multiple types
+yacckind: Grmtools
+grammar: |
+    %parse_param <'a>(_a_count: u64, _b_count: u64)
```

Comment:
I *think* I might know the answer to this question but why the initial `_` on the variable names? Is it because of unused variable warnings in the generated code for actions that don't make use of them?